### PR TITLE
adding rc release for 2.4

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by ehelms** [comment]
Do we not also want to do this for the katello meta RPM?

**Comment by jlsherrill** [comment]
updated

File: katello/katello.spec [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 Name:       katello
 Version:    2.4.0
-Release:    6.nightly%{?dist}
+Release:    7.rc1%{?dist}
```
Comment (ehelms) [inline katello/katello.spec]: 1.rc1

Comment (jlsherrill) [inline katello/katello.spec]: i'd rather not go from 6.nightly to 1.

**Comment by ehelms** [comment]
Need to bump the katello-repos package as well

**Comment by ehelms** [comment]
I hope you are taking notes or updating the release guide as you find these new changes that have to be made since moving to kpackaging for a release :)

**Comment by jlsherrill** [comment]
@ehelms i am taking notes :)

**Comment by jlsherrill** [comment]
@ehelms repos bumped

File: repos/katello-repos.spec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Name:           katello-repos
 Version:        2.4.0
-Release:        2.nightly%{?dist}
+Release:        1.rc1%{?dist}
```
Comment (ehelms) [inline repos/katello-repos.spec]: Following your previous comment, shouldn't this be 3.rc1 ? Wouldn't the same apply for going from 1.nightly to 2.rc1 ?  Not intending to nitpick but rather understand, get it right and ensure consistency.

Comment (jlsherrill) [inline repos/katello-repos.spec]: no you are correct, i totally missed that!  :+1:

Comment (jlsherrill) [inline repos/katello-repos.spec]: updated

File: rubygem-katello/rubygem-katello.spec [hunk 3]
```diff
@@ -13,7 +13,7 @@ Name:    %{?scl_prefix}rubygem-%{gem_name}
 Summary: Katello
 
 Version: 2.4.0
-Release: 1.nightly%{dist}
+Release: 1.rc1%{dist}
```
Comment (ehelms) [inline rubygem-katello/rubygem-katello.spec:16]: I asked this in the previous comment, should this and the installer get bumped to 2.rc1 ?

Comment (jlsherrill) [inline rubygem-katello/rubygem-katello.spec:16]: no IMHO, as 1.nightly is upgradable to 1.rc1

Comment (ehelms) [inline rubygem-katello/rubygem-katello.spec:16]: Again not trying to be picky, just understanding, so 6.nightly could have
been bumped to 6.rc1 ?

Comment (jlsherrill) [inline rubygem-katello/rubygem-katello.spec:16]: True!  Will change to 6.rc1

**Comment by ehelms** [comment]
ACK

**Pull request merged**